test/web_socket_task_test.rb

```diff
@@ -66,262 +59,357 @@
         end
     end
 
-    it "accepts a first connection" do
-        assert websocket_running?(@websocket)
-    end
-
-    describe "the behavior on new connections when pending" do
+    describe "no maximum time since messages" do
```

Comment:
For the unit tests I added a new "describe" level to test with/without filtering, so thats why there are so many changes in this file.

Reply:
I suggest hiding whitespace to review this part.